File: SoloLearn/Carrot Cake/CarrotCake.c
#include <stdio.h>

int main() {
    int carrots, boxes, remainder, missing;
    scanf("%d", &carrots);
    scanf("%d", &boxes);
    remainder = carrots % boxes;
    missing = 7 - remainder
    if (missing <= 0)
        printf("Cake Time");
    else
        printf("I need to buy %d more", missing);
    return 0;
}

File: Code Coach/Carrot Cake/CarrotCake.java
import java.util.Scanner;

public class CarrotCake {
    
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int carrots = input.nextInt();
        input.nextLine();
        int boxes = input.nextInt();
        int remainder = carrots % boxes, missing = 7 - remainder;
        if (missing <= 0)
            System.out.println("Cake Time");
        else    
            System.out.println("I need to buy " + missing + " more");
        input.close();
    }
}
